docs/core/diagnostics/logging-and-tracing.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
file names shouldn't have small words such as and (see https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files)

so please rename this to logging-tracing.md